2012-03-31  Carlos O'Donell  <carlos@systemhalted.org>

	* ports/sysdeps/unix/sysv/linux/hppa/syscalls.list: Add prlimit64
	syscall.

---
 ports/sysdeps/unix/sysv/linux/hppa/syscalls.list |    1 +
 1 file changed, 1 insertion(+)

--- a/ports/sysdeps/unix/sysv/linux/hppa/syscalls.list
+++ b/ports/sysdeps/unix/sysv/linux/hppa/syscalls.list
@@ -35,3 +35,4 @@
 
 setrlimit	-	setrlimit	i:ip	__setrlimit	setrlimit	
 getrlimit	-	getrlimit	i:ip	__getrlimit	getrlimit	
+prlimit64	EXTRA	prlimit64	i:iipp	prlimit64
